libs/pageserver_api/src/shard.rs

```diff
@@ -43,7 +43,7 @@ use utils::id::TenantId;
 pub struct ShardNumber(pub u8);
 
 #[derive(Ord, PartialOrd, Eq, PartialEq, Clone, Copy, Serialize, Deserialize, Debug, Hash)]
-pub struct ShardCount(u8);
+pub struct ShardCount(pub u8);
```

Comment:
This was intentionally private, because callers might otherwise read it expecting the value to be a literal number of shards, whereas it's actually 0 in the case of legacy (1 shard) tenants.  See ShardCount::count for getting the literal number of shards.

Reply:
Ah, thanks. I actually looked for a getter. Idk how I didn't find it :shrug: 
[10d7580](https://github.com/neondatabase/neon/pull/8099/commits/10d7580d22be852fb69dbcb4cd8a571c74052e99)